sidebars.js

```diff
@@ -547,9 +539,32 @@ const sidebars = {
         },
         {
           type: "doc",
-          id: "weaviate/config-refs/distances",
+          id: "weaviate/config-refs/vectorization",
           className: "sidebar-item",
         },
+        {
+          type: "category",
+          label: "Indexing",
+          className: "sidebar-item",
+          collapsed: true,
+          items: [
+            {
+              type: "doc",
+              id: "weaviate/config-refs/indexing/inverted-index",
+              className: "sidebar-item",
+            },
+            {
+              type: "doc",
+              id: "weaviate/config-refs/indexing/vector-index",
+              className: "sidebar-item",
+            },
+            {
+              type: "doc",
+              id: "weaviate/config-refs/distances",
```

Comment:
Since distances are unique to vectors, I wonder if it could be under the vector indexing page? 

Alternatively, as distances also apply to the whole collection outside of the indexing - could be on the top level. Wdyt?

Reply:
I agree, it is back at the top level now.